frontend/src/static/js/v_summary.js

```diff
@@ -176,7 +176,7 @@ window.vSummary = {
       const { REPOS } = window;
       const urlString = repo.location;
 
-      if (urlString.startsWith('http://github.com/') || urlString.startsWith('https://github.com/')) {
+      if (urlString.includes('http://github.com/') || urlString.includes('https://github.com/')) {
```

Comment:
actually what i meant was., can just do a `urlString.includes('github.com')`.
There is too many combination and you cant possibly cover all cases. 
E.g. it is also legit for ppl to misconfigure it with `www` prefix and RepoSense will still be able to clone that.

Reply:
Should we also capture `://` in the search string? `github.com` can be a legit folder name.

Reply:
That make sense. My idea of differentiating local filepath and url (in another issue) is to actually indicate it in the json.

However, another way to go about it is to check if the `organization` is set. Local git has no organization.